frontend/src/Components/HomePage/Profile/Profile.css

```diff
@@ -0,0 +1,14 @@
+.dropdown-menu a{
+   color: var(--smoke);
+}
+
+.dropdown-menu a:hover{
+   background-color: var(--smoke) !important;
```

Comment:
same here, try to add some more specific selectors if needed. https://www.w3schools.com/cssref/css_selectors.asp